### PR TITLE
ES16-108 Add option to xlbridge so that it can change to upcase boole…

### PR DESCRIPTION
**Pull request by jparragap**
…an values if required

### CODE REVIEW

File: lib/XLBridge/excel_report/cell.rb [hunk 1]
```diff
@@ -18,11 +18,18 @@ def initialize(sheet, row_index, column, master_index)
...
       def populate(value, options = {})
         cell.change_contents(value) unless skip_columns(options).include?(column)
+        upcase_boolean(value) if options['upcase_booleans']
         format(options)
       end
 
       private
 
+      def upcase_boolean(value)
+        if [true, false].include? value
```
Comment (royGap) [inline lib/XLBridge/excel_report/cell.rb]: could we extract the boolean verification into another method?
`cell.change_contents(value.to_s.upcase) if boolen?(value)`

File: lib/XLBridge/excel_report/cell.rb [hunk 2]
```diff
@@ -18,11 +18,18 @@ def initialize(sheet, row_index, column, master_index)
...
+          cell.change_contents(value.to_s.upcase)
```
Comment (andresgap) [inline lib/XLBridge/excel_report/cell.rb]: ```cell.change_contents(value.to_s.upcase) if [true, false].include?(value)```

File: lib/XLBridge/excel_report/cell.rb [hunk 3]
```diff
@@ -18,11 +18,18 @@ def initialize(sheet, row_index, column, master_index)
 
       def populate(value, options = {})
         cell.change_contents(value) unless skip_columns(options).include?(column)
+        upcase_boolean(value) if options['upcase_booleans']
```
Comment (andresgap) [inline lib/XLBridge/excel_report/cell.rb]: In the previous line the value is not change if we skip the column, please take that into consideration too. We should not upcase the boolean in that case.

File: lib/XLBridge/excel_report/cell.rb [hunk 4]
```diff
@@ -17,12 +17,19 @@ def initialize(sheet, row_index, column, master_index)
...
+          cell.change_contents(value)
+          upcase_boolean(value) if options['upcase_booleans']
+        end
         format(options)
       end
 
       private
 
+      def upcase_boolean(value)
+        cell.change_contents(value.to_s.upcase) if [true, false].include? value
```
Comment (andresgap) [inline lib/XLBridge/excel_report/cell.rb]: please create a method called boolean?
```def boolean?(value)
  [true, false].include?(value)
end```

**Pull request merged**